persistence/src/commonTest/kotlin/com/wire/kalium/persistence/GlobalDBBaseTest.kt

```diff
@@ -18,9 +18,9 @@
 
 package com.wire.kalium.persistence
 
-import com.wire.kalium.persistence.db.GlobalDatabaseProvider
+import com.wire.kalium.persistence.db.GlobalDatabaseBuilder
 
 expect abstract class GlobalDBBaseTest() {
```

Comment:
This one in particular,  I would prefer to keep it as is, since it is out of scope from the changes, and will require more files added to the change, that are already quite a lot 😓